### PR TITLE
imported and customized aragon/ui DropDown

### PR DESCRIPTION
**Pull request by rkzel**
Customization wasn't especially extensive - all that as needed was chaging some hardcoded styling to make open list of options be dependent on Modal size instead of position of DropDown component itself.

Now, it is understood that taking and customizing components is not the most correct of ways, and it is temporary solution before, I guess, aragon/ui DropDown can accept different styling imposed by our code. Probably. Please let me know what you think...

### CODE REVIEW

File: src/client/components/modals/index.js [hunk 1]
```diff
@@ -146,36 +146,42 @@ const UserInfoModal = ({ ethereumAddress, onSignatures }) => {
...
-      {modal.type === '3boxState' && (
-        <BoxState
-          messageSigning={boxes[ethereumAddress].messageSigning}
-          signaturesRequired={modal.sigsRequired}
-        />
-      )}
+      <div
+        css={`
+          position: relative;
+        `}
```
Comment (chadoh) [inline src/client/components/modals/index.js]: <div css="position: relative">

one line seems easier to read in this case

**Review by chadoh — CHANGES_REQUESTED** [review]
See a tiny change request below, but more than that I'd love before & after screenshots in the description here, and a comment on the specific lines that diverged from aragonUI's implementation.

In general, I'm feeling like fixing this here was a short-term approach, and I suspect our efforts would have been better spent upgrading aragonUI itself. `DropDownItems` probably needs to be inserted at the bottom of `<body>` using `React.Portal`, so it can be absolutely positioned over everything else on the page. This would solve the issue in the general case, for all of our apps.

Do we know if anyone is already working on such a fix for aragonUI?

**Comment by rkzel** [comment]
## before

dropdown-before

## after

dropdown-after

## removing this workaround in the future

To remove/replace DropDown components in the future it is enough to update the following in `readOrEditFields`:

`export { default as DateDropDown } from './DateDropDown'`

DateDropDown is self-contained and only slightly changed component taken from Aragon UI.

**Review by chadoh — APPROVED** [review]
Thanks for the before & after shots, and for explaining how to remove. 

Looks like an acceptable workaround for now.

Befor merging, please remove the `package-lock.json` change. I don't see an accompanying `package.json` change, so I don't think it should be here.

**Pull request merged**